components/layout/pageLayout/SubNavbar.tsx

```diff
@@ -20,7 +20,12 @@ const INDENTATION = 16;
 export default function SubNavbar({ currentTab }: { currentTab: TreeNode }) {
   const t = useTranslations('Nav');
   const rootTab = getRootTab(currentTab as SegmentNode);
-  const subTabs = getAllSubTabs(rootTab);
+  const subTabs =
+    // 학생회 하위 탭은 서브내비 노출 X
+    rootTab === community
+      ? getAllSubTabs(rootTab).filter((tab) => tab.parent !== council)
+      : getAllSubTabs(rootTab);
+
```

Comment:
아래처럼하는게 목적이 더 명확해보이는데 어때? 성능 차이도 별로 없을 것 같아서!

```suggestion
  const subTabs = getAllSubTabs(rootTab).filter((tab) => tab.parent !== council)
```